### PR TITLE
chore: add wiki to `.github/docs/wiki`

### PR DESCRIPTION
**Pull request by NiceAesth**
# Describe your changes

## Checklist
- [x] I've manually tested my code
- [x] The changes pass pre-commit checks (`make lint`)
- [x] The changes follow coding style

### CODE REVIEW

File: .github/docs/wiki/6.-Post-Set-Up.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <li>c6.yourdomain.com</li>
+            <li>ce.yourdomain.com</li>
+            <li>cho.yourdomain.com</li>
+            <li>c.yourdomain.com</li>
+            <li>yourdomain.com</li>
+            <li>i.yourdomain.com</li>
+            <li>map.yourdomain.com</li>
+            <li>osu.yourdomain.com</li>
+            <li>s.yourdomain.com</li>
+            <li>web.yourdomain.com</li>
```
Comment (7mochi) [inline .github/docs/wiki/6.-Post-Set-Up.md:57]: [this changes](https://github.com/osuAkatsuki/bancho.py/commit/488b7c9ca33fa539720faf0eaa792d8af1b7af9d) are missing

Comment (NiceAesth) [reply .github/docs/wiki/6.-Post-Set-Up.md:57]: Yeah, I know. Was intending to do that in a different PR, but I guess I can do that here.

Comment (NiceAesth) [reply .github/docs/wiki/6.-Post-Set-Up.md:57]: Anything else missing?

Comment (7mochi) [reply .github/docs/wiki/6.-Post-Set-Up.md:57]: other than that lgtm

**Review by 7mochi — APPROVED** [review]
lgtm

**Pull request merged**